play/src/front/Chat/Components/Room/RoomTimeline.svelte

```diff
@@ -17,17 +18,36 @@
     let autoScroll = true;
     let onScrollTop = false;
 
+    let oldScrollHeight = 0;
+
     let scrollTimer: ReturnType<typeof setTimeout>;
+    $: isLoadingMessages = false;
```

Comment:
```suggestion
    let isLoadingMessages = false;
```
The `$:` is probably unnecessary. All variables at the root level of the script are reactive.
The "$" is useful to recompute automatically depending on the right part of the equal. In your case, it never changes because it is "false".